app/decorators/physical_server_decorator.rb

```diff
@@ -0,0 +1,22 @@
+class PhysicalServerDecorator < MiqDecorator
+  def fileicon
+    "svg/vendor-#{vendor.downcase}.svg"
+  end
+
+  def fonticon
```

Comment:
As this method contains a static string, it is better to declare it as `self.fonticon`.